integration/test_integration_basics.py

```diff
@@ -192,15 +192,15 @@ def test_h2_tls_context_configuration(self):
     """
     Verifies specifying tls cipher suites works with the h2 pool
     """
-    parsed_json = self.runNighthawkClient([
+    parsed_json, logs = self.runNighthawkClient([
         "--duration 1",
         "--tls-context {common_tls_context:{tls_params:{cipher_suites:[\"-ALL:ECDHE-RSA-AES128-SHA\"]}}}",
         self.getTestServerRootUri()
     ])
     counters = self.getNighthawkCounterMapFromJson(parsed_json)
     self.assertEqual(counters["ssl.ciphers.ECDHE-RSA-AES128-SHA"], 1)
 
-    parsed_json = self.runNighthawkClient([
+    parsed_json, logs = self.runNighthawkClient([
```

Comment:
If logs aren't going to be used, you make this `parsed_json, _ = self.runNight...`